src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/AutoCrafterListener.java

```diff
@@ -38,6 +45,11 @@ public AutoCrafterListener(SlimefunPlugin plugin) {
     @EventHandler
     public void onInteract(PlayerRightClickEvent e) {
         Optional<Block> clickedBlock = e.getClickedBlock();
+        List<Recipe> recipes = org.bukkit.Bukkit.getRecipesFor(e.getItem());
```

Comment:
Import it and call `Bukkit` class directly.

Reply:
I don't know why... But importing org.bukkit.Bukkit doesn't work. Only doing a static import of org.bukkit.Bukkit.getRecipesFor works. But I think static import is not allowed right...? What should I do?

Reply:
When you try to import this, plugin not compiling or what?

Reply:
Yeah, it's not compiling...